pkgs/top-level/python-aliases.nix

```diff
@@ -62,6 +62,9 @@ mapAliases ({
   Keras = keras; # added 2021-11-25
   lammps-cython = throw "lammps-cython no longer builds and is unmaintained"; # added 2021-07-04
   MechanicalSoup = mechanicalsoup; # added 2021-06-01
+  nmigen-boards = amaranth-boards; # added 2022-01-02
+  nmigen = amaranth; # added 2022-01-02
+  nmigen-soc = amaranth-soc; # added 2022-01-02
```

Comment:
nMigen is our registered trademark and those aliases arguably constitute infringement.